src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -49,7 +49,7 @@ def loadYangModel(self):
             self.yangFiles = [f.split('/')[-1] for f in self.yangFiles]
             self.yangFiles = [f.split('.')[0] for f in self.yangFiles]
             self.sysLog(syslog.LOG_DEBUG,'Loaded below Yang Models')
-            self.sysLog(syslog.LOG_DEBUG,self.yangFiles)
+            self.sysLog(syslog.LOG_DEBUG,str(self.yangFiles))
```

Comment:
Will it be possible for you to paste  the output after fix in description or here, Thx

Reply:
[breakout_UT.txt](https://github.com/Azure/sonic-buildimage/files/7894213/breakout_UT.txt)